src/components/swap/SwapDetailsDropdown.test.tsx

```diff
@@ -0,0 +1,52 @@
+import { TEST_ALLOWED_SLIPPAGE, TEST_TOKEN_1, TEST_TRADE_EXACT_INPUT, toCurrencyAmount } from 'test-utils/constants'
+import { fireEvent, render, screen } from 'test-utils/render'
+
+import SwapDetailsDropdown from './SwapDetailsDropdown'
+
+jest.mock('@web3-react/core', () => {
+  const web3React = jest.requireActual('@web3-react/core')
+  return {
+    ...web3React,
+    useWeb3React: () => ({
+      chainId: 1,
+    }),
+  }
+})
+
```

Comment:
This is no longer necessary - it is mocked in `setupTests`.
You'll need to remock it (as `mocked(useWeb3React).mockReturnValue({...})`), but only if you want to mock a connected account.

Reply:
ty done